cookbooks/swift/files/default/etc/swift/proxy-server/proxy-noauth.conf.d/20_settings.conf

```diff
@@ -7,6 +7,7 @@ pipeline = catch_errors gatekeeper healthcheck proxy-logging cache list-endpoint
 [filter:gatekeeper]
 use = egg:swift#gatekeeper
 shunt_inbound_x_timestamp = false
+allow_reserved_names_header = true
```

Comment:
not sure this is strictly necessary, I have been playing with https://github.com/swiftstack/vagrant-swift-all-in-one/pull/88

I think if upstream gatekeeper was more malleable to development we'd have some better options